default argument handling for mobile unboxing codegen

bdhirsh opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #78353

Differential Revision: [D36702695](https://our.internmc.facebook.com/intern/diff/D36702695)

bdhirsh committed:
default argument handling for mobile unboxing codegen

[ghstack-poisoned]

larryliu0820 reviewed (approved):
Thanks for the quick help!

bdhirsh committed:
Update on "default argument handling for mobile unboxing codegen"


Differential Revision: [D36702695](https://our.internmc.facebook.com/intern/diff/D36702695)

[ghstack-poisoned]
